airbyte-integrations/connectors/source-amplitude/acceptance-test-docker.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env sh
 
 # Build latest connector image
-docker build . -t $(cat acceptance-test-config.yml | grep "connector_image" | head -n 1 | cut -d: -f2)
+docker build . -t $(cat acceptance-test-config.yml | grep "connector_image" | head -n 1 | cut -d: -f2-)
 
-# Pull latest acctest image
+# Pull latest acceptance test image
```

Comment:
nit: Can we either not make this change or apply it to all connectors (in a separate PR) for the sake of consistency?